specification/monitor/resource-manager/readme.md

```diff
@@ -27,16 +27,27 @@ These are the global settings for the MonitorClient API.
 ``` yaml !$(python) || !$(track2)
 title: MonitorClient
 ```
+
 ``` yaml $(python) && $(track2)
 title: MonitorManagementClient
 ```
 
 ``` yaml
 description: Monitor Management Client
 openapi-type: arm
-tag: package-2020-03
+tag: package-preview-2017-09
```

Comment:
The tag used here should contain all of the files for monitor
```suggestion
tag: package-2020-03
```

Reply:
Thanks for reviewing this and catching this, I've incorporated your feedback.